src/views/NewSettings/components/Sections/Workspace/Logs/AccessLogsSection.tsx

```diff
@@ -60,7 +60,7 @@ export const AccessLogsSection = ({ onClosePreferences }: LogsViewProps): JSX.El
   );
 
   const handleActivityFilters = (actionType: WorkspaceLogType) => {
```

Comment:
same with this `hooks`, could added it in `Logs` folder inside `NewSettings/hooks` for example in order to maintain it separated for other `hooks`, but inside hooks directory in newSettings